server/aws-lsp-codewhisperer/src/language-server/netTransform/transformHandler.ts

```diff
@@ -289,10 +290,17 @@ export class TransformHandler {
                     break
                 }
             } catch (e: any) {
-                const errorMessage = (e as Error).message ?? 'Error in GetTransformation API call'
-                this.logging.log('CodeTransformation: GetTransformation error = ' + errorMessage)
-                status = 'FAILED'
-                break
+                if (status === 'Retry' && getTransformRetry === getTransformMaxRetry) {
+                    const errorMessage = (e as Error).message ?? 'Error in GetTransformation API call'
+                    this.logging.log('CodeTransformation: GetTransformation error = ' + errorMessage)
+                    status = 'FAILED'
```

Comment:
Can we enumerate the polling status values? So far, I've seen:
* Retry
* FAILED
* Timed_out
* NOT_FOUND